src/services/searchnearby/index.js

```diff
@@ -1,8 +1,8 @@
-const hooks = require('./hooks');
 const Promise = require('bluebird');
 const errors = require('feathers-errors');
-const Pin = require('../pin/pin-model.js');
+const hooks = require('./hooks');
```

Comment:
Wanna separate internal dependencies from external dependencies?

Also just notice `Promise` should come after `errors`